scripts/tests/test_util.py

```diff
@@ -77,3 +80,22 @@ def test_node_desc_fail(name):
 )
 def test_to_hostlist_fast(names, expected):
     assert util.to_hostlist_fast(names.split(",")) == expected
+
+
+@pytest.mark.parametrize(
+    "api,expected",
+    [
+        (util.ApiEndpoint.BQ, True),
```

Comment:
Why not make `expected` a proper instance of `ClientOptions`, so this test actually tests logic

Reply:
Hopefully the change I made for this is what you meant.  Let me know and I can change it later on in another PR.